src/app/layout.tsx

```diff
@@ -45,7 +45,7 @@ export default function RootLayout({ children }: RootLayoutProps) {
 
 				{/* renamed Huy's bg-webdev-temp to rootBackground */}
 				{children}
-				<MLHBanner></MLHBanner>
+				{/* <MLHBanner></MLHBanner> */}
```

Comment:
The MLH banner must always be on the website 
```suggestion
				<MLHBanner></MLHBanner>
```